llvmlite/tests/test_binding.py

```diff
@@ -1989,5 +2011,21 @@ def rel():
             llvm.ffi.unregister_lock_callback(acq, rel)
 
 
+@unittest.skipUnless(platform.machine().startswith('x86'), "only on x86")
```

Comment:
Can you add tests for AArch64 and PPCLE64 (if they're relevant), as I believe they're they're tested on the buildfarm, and would demonstrate that it works on platforms other than x86?

Reply:
I can try my best to get these done today after school.

Reply:
There;'s no hurry - please feel free to work through this in your own time.